drake/systems/plants/@RigidBodyManipulator/signedDistances.m

```diff
@@ -0,0 +1,25 @@
+function [phi,n,x,body_x, body_idx] = signedDistances(obj,kinsol,points,use_margins)
```

Comment:
i don't like the name here.  the method is very similar to collisionDetect (which also returns the signedDistances).  The only real difference is that you are doing queries from these new points to every collsion object in the collision world, as opposed to returning the distances between the existing objects in the collision world.  The method signature and documentation should make this much more clear.